snippets/react-snippets.code-snippets

```diff
@@ -207,5 +207,22 @@
             ""
         ],
         "description": "Use `role='switch'` on checkboxes that represent an 'on' or 'off' state. Add `aria-checked` to indicate whether component is on or off. Add `aria-required` if the field is required."
-    }
+    },
+    "SearchboxAria" : {
+        "prefix": [
+            "SearchboxAria"
+        ],
+        "body": [
+            "role='searchbox'",
+            "aria-activedescendant='$1'",
```

Comment:
based on this thread: https://github.com/w3c/aria-practices/issues/1903 i decided to include the existing properties provided by the textbox snippet, since the only difference between this and a textbox seems to be `<input type="search">` vs `<input type="text">`.

Reply:
✨